db/blob/blob_file_reader_test.cc

```diff
@@ -217,12 +277,42 @@ TEST_F(BlobFileReaderTest, CreateReaderAndGetBlob) {
     PinnableSlice value;
     uint64_t bytes_read = 0;
 
-    ASSERT_TRUE(reader
-                    ->GetBlob(read_options, shorter_key,
-                              blob_offset - (sizeof(key) - sizeof(shorter_key)),
-                              blob_size, kNoCompression, &value, &bytes_read)
-                    .IsCorruption());
+    ASSERT_TRUE(
+        reader
+            ->GetBlob(read_options, shorter_key,
+                      blob_offsets[0] - (keys[0].size() - sizeof(shorter_key)),
```

Comment:
I think technically there's an off-by-one issue here: I think the correct adjustment would be `keys[0].size() - (sizeof(shorter_key) - 1)`. (Earlier, when both keys were stored in `char` arrays, the two -1's canceled each other out.)

Not sure if it makes a difference though

Reply:
Done